GitCommands/Git/GitModule.cs

```diff
@@ -2172,6 +2172,53 @@ public IReadOnlyList<GitStash> GetStashes(bool noLocks = false)
             return stashes;
         }
 
+        public async Task<ExecutionResult> GetSingleDifftoolAsync(
+            ObjectId? firstId,
+            ObjectId? secondId,
+            string? fileName,
+            string? oldFileName,
+            ArgumentString extraDiffArguments,
+            bool cacheResult,
+            bool isTracked,
+            bool useGitColoring,
+            CancellationToken cancellationToken)
+        {
+            // fix refs slashes
+            fileName = fileName.ToPosixPath();
+            oldFileName = oldFileName.ToPosixPath();
+            string? firstRevision = firstId?.ToString().ToPosixPath();
+            string? secondRevision = secondId?.ToString().ToPosixPath();
+
+            string? diffOptions = _revisionDiffProvider.Get(firstRevision, secondRevision, fileName, oldFileName, isTracked);
+
+            GitArgumentBuilder args = new("difftool", commandConfiguration: null, "--no-pager")
+            {
+                "--find-renames",
+                "--find-copies",
+                "-y",
+                extraDiffArguments,
+                diffOptions
+            };
+
+            CommandCache? cache = false && cacheResult
```

Comment:
can't never be true

Reply:
changing, cacheResult is in addition always false now (cache key needs env var)
But keeping the check